sharktuner/build_tools/build_linux_package.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
```

Comment:
Changes to this file should go to a separate PR that enables nightly releases for sharktuner.